README.md

```diff
@@ -62,6 +62,32 @@ That's ok. `Quandl::Config` will find your config file if it's in a `config` fol
 
 If you want to provide the environment, use `ENV['RAILS_ENV']` or `ENV['RAKE_ENV']`. If you don't, the `default` environment will be assumed.
 
+### How do I override default values?
+
+You can override default values by adding a `configuration_options` method to your class/instance. The options available to be overwritten are:
+
+* `root_path` - Override this when you want to change the default path to your configuration files root folder. (default: your project path)
```

Comment:
Can you also update the section above, `What if my project is not a Rails app?`? That now gives outdated advice :smile_cat: